tools/docker/Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM alpine:3.5
-
-MAINTAINER Mark Harrison <mark@mivok.net>
-
-RUN apk add --no-cache ruby && \
-    gem install --no-rdoc --no-ri mdl && \
+FROM alpine:3.11.3
+RUN echo "mdl:x:1001:mdl" >> /etc/group && \
+    echo "mdl:x:1001:1001:mdl,,,:/home/mdl:/sbin/nologin" >> /etc/passwd && \
+    apk add --no-cache ruby && \
```

Comment:
It seems like `--no-rdoc` and `--no-ri` would be desired in a docker container... why are you dropping them?

Reply:
These options are not available anymore in recent rubygem versions:

```
  Deprecated Options:
        --[no-]rdoc                  Generate RDoc for installed gems
                                     Use --document instead
        --[no-]ri                    Generate ri data for installed gems.
                                     Use --document instead
    -u, --[no-]update-sources        Update local source cache
```

I will try to find equivalent options.

Reply:
According to the help menu, these options can be omitted as one has to explicitly define whether one want to install rdoc and ri documentation:

```
  Install/Update Options:
    -i, --install-dir DIR            Gem repository directory to get installed
                                     gems
    -n, --bindir DIR                 Directory where executables are
                                     located
        --document [TYPES]           Generate documentation for installed gems
                                     List the documentation types you wish to
                                     generate.  For example: rdoc,ri
```

Reply:
As far as I can tell from the [commit that deprecated them](https://github.com/rubygems/rubygems/commit/0d22d6025ce2653ec9fb20a4e0745c79bda677da), the default for `--document` is `ri,rdoc`. So I think you want `--no-document` (which seems to be confirmed by various blog posts like [this one](https://www.honeybadger.io/blog/how-to-globally-disable-rdoc-and-ri-during-gem-installs/).

Reply:
@jaymzh You are right. I have added a commit.